portfolio/src/main/java/com/google/sps/data/FindSchedule.java

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public final class FindSchedule {
```

Comment:
Consider adding some class comment. Also, how do you think we can make this class extensible? Consider adding TODO comment if you think it can be made more extensible, as we add scheduling algorithms.

Reply:
Also does this class have any instance variable? Maybe it can be a static class, as well as all of its methods.

Reply:
I left a TODO to refactor it into an abstract class for the next PR and made it static.